mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb

```diff
@@ -1536,12 +1536,7 @@ func TestAccContainerCluster_withNodeConfig(t *testing.T) {
 	})
 }
 
-// Note: Updates for these are currently known to be broken (b/361634104), and
-// so are not tested here.
-// They can probably be made similar to, or consolidated with,
-// TestAccContainerCluster_withInsecureKubeletReadonlyPortEnabledInNodeConfigUpdates
-// after that's resolved.
-func TestAccContainerCluster_withNodeConfigKubeletConfigSettings(t *testing.T) {
+func TestAccContainerCluster_withNodeConfigKubeletConfigSettingsUpdates(t *testing.T) {
```

Comment:
I confirmed that with the test changes in this PR, but not the code changes, the test fails successfully (without explicit resource checks on individual fields):

```
    resource_container_cluster_test.go:1369: Step 3/6 error: After applying this test step, the plan was not empty.
        stdout:
        
        
        Terraform used the selected providers to generate the following execution
        plan. Resource actions are indicated with the following symbols:
          ~ update in-place
        
        Terraform will perform the following actions:
        
          # google_container_cluster.with_node_config_kubelet_config_settings will be updated in-place
          ~ resource "google_container_cluster" "with_node_config_kubelet_config_settings" {
                id                                       = "projects/zelig-tfprovidertest-sandbox/locations/us-central1-f/clusters/tf-test-cluster-r77hrrgh36"
                name                                     = "tf-test-cluster-r77hrrgh36"
                # (30 unchanged attributes hidden)
        
              ~ node_config {
                    tags                        = []
                    # (17 unchanged attributes hidden)
        
                  ~ kubelet_config {
                      ~ cpu_cfs_quota                          = false -> true
                      + cpu_cfs_quota_period                   = "100ms"
                      + cpu_manager_policy                     = "none"
                      ~ pod_pids_limit                         = 0 -> 2048
                        # (1 unchanged attribute hidden)
                    }
        
                    # (1 unchanged block hidden)
                }
        
                # (18 unchanged blocks hidden)
            }
        
        Plan: 0 to add, 1 to change, 0 to destroy.
 --- FAIL: TestAccContainerCluster_withNodeConfigKubeletConfigSettingsUpdates (729.60s)
```

![image](https://github.com/user-attachments/assets/573b8c77-6da7-4cb5-9f28-5b2089eb5a47)